src/Languages/Html/HtmlLanguage.php

```diff
@@ -20,6 +23,9 @@ public function getInjections(): array
             new PhpShortEchoInjection(),
             new CssInjection(),
             new CssAttributeInjection(),
+            new TwigPropertyInjection(),
```

Comment:
When  I do this, I get infinite loop (twig language want inject twig language for ever). Because twig regex doesnt have outside tags. `(?<match>({{(.|\n)*?}}))` . and this is because I want highlight open and close tags like `{#` and `#}`.

Reply:
I'll merge this in as-is, and I'll tinker with it some more afterwards 👍